bin/lib/formatContentForReading.js

```diff
@@ -31,7 +31,7 @@ const REMOVALS = [
 ];
 const REMOVALS_PATTERN = `\\b(${REMOVALS.join('|') })\\b`;
 const REMOVALS_REGEXP = new RegExp(REMOVALS_PATTERN, 'ig');
-debug(`REMOVALS_REGEXP=${REMOVALS_REGEXP}`);
+// debug(`REMOVALS_REGEXP=${REMOVALS_REGEXP}`);
```

Comment:
If the debugs are no longer being used, can we remove them? Or should we leave them on for the prod server?